src/main/java/thePackmaster/cards/Vexed.java

```diff
@@ -0,0 +1,37 @@
+package thePackmaster.cards;
+
+import com.evacipated.cardcrawl.mod.stslib.fields.cards.AbstractCard.SoulboundField;
+import com.megacrit.cardcrawl.actions.common.ExhaustSpecificCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.patches.odditiespack.PackmasterFoilPatches;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class Vexed extends AbstractPackmasterCard {
```

Comment:
Note that this being unplayable is taken care of by the default logic for curses in `AbstractCard.canUse`.